src/NJsonSchema/NJsonSchema.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard2.0;net40;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
This is a bit a problem as i know there are ppl using this lib with netstd < 2.0 and also with .NET 4.0
Is this really required?

Reply:
I restored the old targets, seems that we can keep this project targeting as-is. Good catch.